cypress-tests/cypress/fixtures/confirm-body.json

```diff
@@ -1,20 +1,10 @@
 {
   "client_secret": "",
   "return_url": "https://hyperswitch.io",
-  "payment_method": "card",
   "confirm": true,
-  "payment_method_data": {
-    "card": {
-      "_comment": "these details are fetched from utils",
-      "card_number": "YourCardNumberHere",
-      "card_exp_month": "04",
-      "card_exp_year": "24",
-      "card_holder_name": "xyz",
-      "card_cvc": "424",
-      "card_issuer": "",
-      "card_network": "Visa"
-    }
-  },
+  "payment_method": "card",
+  "payment_method_type": "credit",
+  "payment_method_data": {},
```

Comment:
This has been made empty to allow bank_transfers to work. Have verified with cards, it **does not** harm them.